docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -9,88 +9,259 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The following steps guide you through configuring your JS application to automatically upload sourcemap files during the project build. Additionally, it explains how to set the appropriate ID in your Backtrace client to match errors with the corresponding sourcemap files.
+The following steps guide you through configuring your JS application to automatically upload sourcemap files during the project build.
 
 ## What You'll Need
 
 - A Backtrace account ([log in](https://backtrace.io/login) or sign up for a [free trial license](https://backtrace.io/sign-up)).
 - Your subdomain name (used to connect to your Backtrace instance). For example, `https://example-subdomain.sp.backtrace.io`.
-- A Backtrace project and a submission token.
+- A Backtrace project, an error submission token and a symbol submission token.
 
-:::tip Generate a Submission Token
+:::tip Generating Tokens
+## Generating an Error Submission Token
 
 1. In the Backtrace Console, go to **Project settings > Error submission > Submission tokens**.
 1. Select **+**.
 
+
+## Generating a Symbol Submission Token
+
+1. In the Backtrace Console, go to **Project settings > Symbols > Access tokens**.
+1. Select **+**.
+
```

Comment:
@perf2711 - Can we direct the user to Settings/Project/API Tokens? The UI team plans to deprecate the dedicated pages for Symbol and Submission tokens.

Reply:
Resolved with your PR.